contracts/badger-bridge/BadgerBridgeAdapter.sol

```diff
@@ -163,6 +176,20 @@ contract BadgerBridgeAdapter is OwnableUpgradeable, ReentrancyGuardUpgradeable {
         uint256 startBalanceRenBTC = renBTC.balanceOf(address(this));
         uint256 startBalanceWBTC = wBTC.balanceOf(address(this));
 
+        if (_token == address(ibBTC)) {
```

Comment:
Let's just use a boolean flag here since `_token` has another meaning (used for wbtc -> renbtc swapping). Based on the flag here we can choose which vault token to unwrap to (pre computed and passed in as `_vault` arg). 

We'll need to do some optimal pathing for burns as well as mints
https://github.com/defidollar/ibBTC/blob/main/contracts/peaks/BadgerSettPeak.sol#L105

Pathing calc is not implemented in the zap contract for burns so we can push this off to a follow up PR. 
https://etherscan.io/address/0xC25099792E9349C7DD09759744ea681C7de2cb66#readContract
^ will be a combination of calling `calcRedeem()` on the peak contract and `calc_withdraw_one_coin()` on the curve pool contract and also for the wbtc vault as well (just calcRedeem() I think).